easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb

```
@@ -0,0 +1,494 @@
+easyblock = 'PythonBundle'
+
+name = 'Python-bundle-PyPI'
+version = '2023.06'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages from PyPI"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('UnZip', '6.0'),
+    ('Rust', '1.70.0'),  # required for setuptools-rust, which is needed for cryptography and bcrypt
+    ('pkgconf', '1.9.5'),
+    ('git', '2.41.0', '-nodocs'),  # required for pbr
+    ('setuptools-rust', '1.6.0'),  # required for cryptography and bcrypt
```

The idea is that the bundle does not depend on Rust. It should depend on `cryptography`. If the extension `bcrypt` needs Rust to be build/installed, I would move it into its own easyconfig out of the bundle.

```suggestion
    ('pkgconf', '1.9.5'),
    ('git', '2.41.0', '-nodocs'),  # required for pbr
```

---

Yes, that is a good idea. I separated `bcrypt` and `paramiko` in https://github.com/easybuilders/easybuild-easyconfigs/pull/18208/commits/ff899417ddb7ae3a81e7a8d91cec0bdcf1513865 and https://github.com/easybuilders/easybuild-easyconfigs/pull/18208/commits/00d1fc4ec8440337420d66ee2afcbda8f287db9b